controller/resource_roles.go

```diff
@@ -2,29 +2,42 @@ package controller
 
 import (
 	"context"
+	"fmt"
 	"github.com/fabric8-services/fabric8-auth/app"
 	"github.com/fabric8-services/fabric8-auth/application"
+	permissionservice "github.com/fabric8-services/fabric8-auth/authorization/permission/service"
 	identityrole "github.com/fabric8-services/fabric8-auth/authorization/role/identityrole/repository"
 	roleservice "github.com/fabric8-services/fabric8-auth/authorization/role/service"
 	"github.com/fabric8-services/fabric8-auth/errors"
 	"github.com/fabric8-services/fabric8-auth/jsonapi"
 	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/login"
 	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+)
+
+const (
+	// ROLE_ASSIGNMENT_SCOPE is the scope the user needs to have as part of the roles assigned to her
+	// that will enabled her to assign roles to other users.
+	ROLE_ASSIGNMENT_SCOPE = "assign_role" // The exact name will be refactored in a different PR/commit once we decide on the name of the role.
```

Comment:
Should be "assign_scope" I think. It's a scope. Not a role ;)

Reply:
My understanding is that this scope is being used to allow a user to assign a role to another user for a resource, so `assign_role` would be correct here.

Reply:
Ah, right! Didn't think about in this way ;)